### ISSUE TITLE
Adding an extremely long wallet name makes the page unusable

### ISSUE THREAD

**Issue by creyeknife** [O3Labs/O3Android#267]
If you add a really long wallet name, the password and button get pushed off the screen and it does not scroll

Fix
Limit the number of characters for a wallet name
It might be goo to add scroll to the view as well just in case they have a long name and long passwords and a small screen.

![screenshot_20181127-152611](https://user-images.githubusercontent.com/16552874/49063068-84462200-f259-11e8-80f3-ed1135afb8e3.png)

**Comment by saltyskip** [comment]
Maybe need limit on wallet name cahracters too?

**Comment by creyeknife** [comment]
yeah it messes with the whole app actually. Tested a bit more and the longer names overlap everything and get all whacky. Best to just avoid them big names in.

Portfolio falls over 3 lines and covers most of the top section, the wallet settings view is 2 lines and overlaps buttons and stuff.
